Sources/StreamVideo/Call.swift

```diff
@@ -381,11 +383,25 @@ public class Call: @unchecked Sendable, WSEventsSubscriber {
     ) async throws -> UpdateUserPermissionsResponse {
         try await updatePermissions(
             for: userId,
-            granted: permissions,
+            granted: permissions.map(\.rawValue),
             revoked: []
         )
     }
     
+    @discardableResult
+    public func grant(request: PermissionRequest) async throws -> UpdateUserPermissionsResponse {
+        let response = try await updatePermissions(
+            for: request.user.id,
+            granted: [request.permission],
+            revoked: []
+        )
+        executeOnMain { [weak self] in
+            guard let self else { return }
+            self.state.removePermissionRequest(request: request)
```

Comment:
What will happen if a permissionRequests has been received but another host reply to it?